lib/matplotlib/_type1font.py

```diff
@@ -768,13 +861,262 @@ def transform(self, effects):
                     )
                 len0 += len(value) - pos1 + pos0
 
-        data = bytes(data)
-        return Type1Font((
-            data[:len0],
-            self._encrypt(data[len0:], 'eexec'),
+        return data[:len0], data[len0:]
+
+    def subset(self, characters: Iterable[int]) -> T.Self:
+        """
+        Return a new font that only defines the given characters.
+
+        Parameters
+        ----------
+        characters : sequence of bytes
```

Comment:
of bytes, of ord(bytes) (ASCII codepoints -- note that you typed this as Iterable[int]), or of indices into the encoding array? (e.g. how to interpret this for non-standard encodings?)